sherpa/models/template.py

```diff
@@ -17,6 +18,63 @@
 #  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
+# pylint: disable=invalid-name
+
+"""Use data as model templates.
+
+Allow models to be created from data, where the model data,
+represented as a `sherpa.models.basic.TableModel`, can be associated
+with a set of parameters. The `TemplateModel` class only supports
+access to the parameter values, but the `InterpolatingTemplateModel`
+class adds interpolation of the parameter values (although this is
+class should not be used directly, and the `KNNInterpolator` class
+used instead).
+
+Examples
+========
+
+A set pf three templates are set for the "scale" parameter
```

Comment:
```suggestion
A set of three templates are set for the "scale" parameter
```

Hmmm, should probably add a little-more description here.

Reply:
Done in a rebase.